src/AirtableClient.php

```diff
@@ -111,17 +111,47 @@ public function findTheLatest(string $table, $field, ?string $dataClass = null):
         return AirtableRecord::createFromRecord($recordData);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function addOneRecord(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
+    {
+        $url = sprintf(
+            '%s/%s',
+            $this->id,
+            $table
+        );
+        $response = $this->request($url, 'POST', $fields);
+
+        $recordData = $response->toArray() ?? null;
+
+        if (!$recordData['id']) {
+            return null;
+        }
+
+        if (null !== $dataClass) {
+            $recordData['fields'] = $this->normalizer->denormalize($recordData['fields'], $dataClass);
+        }
+
+        return AirtableRecord::createFromRecord($recordData);
+    }
+
     /**
      * Use the HttpClient to request Airtable API and returns the response.
      */
-    private function request(string $url): ResponseInterface
+    private function request(string $url, string $method, ?array $body = null): ResponseInterface
     {
+        $params = ['auth_bearer' => $this->key];
+
+        if ('POST' === $method) {
+            $params = $params + ['headers' => ['Content-Type' => 'application/json']];
```

Comment:
You use `json` instead of body see https://symfony.com/doc/current/http_client.html#uploading-data
```php
$params[] =  ['headers' => ['Content-Type' => 'application/json']];
$params[] = ['json' => ['fields' => $body]];
```